virtcontainers/sandbox.go

```diff
@@ -1253,12 +1253,6 @@ func (s *Sandbox) CreateContainer(contConfig ContainerConfig) (VCContainer, erro
 		return nil, err
 	}
 
-	// Store it.
-	err = c.storeContainer()
```

Comment:
Why would you delete `storeContainer()` here? If `updateResources` does nothing, we won't have anyone storing container.

Reply:
updated